docs/tequila-api.md

```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
 sidebar_label: What is Tequila API?
 ---
 
-_This page is a stub._
+With the [Tequila](https://tequila.kiwi.com) platform, you can build your o travel business application for free, with the [kiwi.com](https://www.kiwi.com)'s content, technology, and services.
```

Comment:
`You can build your travel business application` without the o

Reply:
Thanks, will change